src/img/icons.json

```diff
@@ -1,6 +1,7 @@
 {
   "icons": [
     "active-task",
+    "add-existing-safe",
```

Comment:
Icon names should be descriptive of the icon itself, not what we're using it for.

With that being said, this is like the 3rd or 4th variation on the plus sign, so naming it would be bad,

But it should be somewhere along the lines of `plus-heavy`